src/styles/blocks/watch.scss

```
@@ -0,0 +1,57 @@
+.watch {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 400px;
+  height: 400px;
+  border: 5px solid grey;
+  background-color: antiquewhite;
+  border-radius: 50%;
+  z-index: 1;
+
+  &__center {
+    position: absolute;
+    display: inline-block;
+    width: 10px;
+    height: 10px;
+    background-color: black;
+    border-radius: 50%;
+    z-index: 4;
+  }
+
+  &__second {
+    position: absolute;
+    bottom: 198px;
+    border: 2px solid red;
+    height: 185px;
+    transform-origin: 50% bottom;
+    animation-name: round;
+    animation-duration: 60s;
+    animation-timing-function: linear;
+    animation-iteration-count: infinite;
+    z-index: 3;
+  }
+
+  &__minute {
+    position: absolute;
+    bottom: 198px;
+    border: 5px solid black;
```

don't forget to create variables for all colors
```suggestion
    border: 5px solid $c-black;
```